dbscripts/xml/upgrade.xml

```diff
@@ -208,6 +208,10 @@
 		<note file="docs/release-notes/README-3.3.0" />
 	</upgrade>
 
+	<upgrade minversion="3.3.0.16" maxversion="3.3.0.19">
```

Comment:
Since this will presumably be included in 3.3.0-19, shouldn't this be `maxversion="3.3.0.18"`?

Reply:
Yes!